From 290efa1cf714c080c32eed34a0fba49206b1b962 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Mon, 24 Jul 2017 10:30:01 +0800
Subject: [PATCH 2270/5242] MLK-16054-2 ARM64: dts: fsl-imx8qm-lpddr4-arm2:
 add OTG1 vbus power pinctrl

commit  0612ea4f53879e39680959e8ce6f1ebb895c84f1 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add OTG1 vbus power pinctrl.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index ca9e8d5..7e0ae18 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -428,6 +428,12 @@
 				SC_P_PCIE_CTRL1_PERST_B_LSIO_GPIO5_IO00		0x00000021
 			>;
 		};
+
+		pinctrl_usbotg1: usbotg1 {
+			fsl,pins = <
+				SC_P_USB_SS3_TC0_CONN_USB_OTG1_PWR		0x00000021
+			>;
+		};
 	};
 };
 
@@ -462,6 +468,8 @@
 };
 
 &usbotg1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_usbotg1>;
 	srp-disable;
 	hnp-disable;
 	adp-disable;
-- 
1.7.9.5

